include/camp/defines.in.hpp

```diff
@@ -106,7 +105,11 @@ namespace camp
 #define CAMP_SUPPRESS_HD_WARN
 #endif
 
-#if defined(ENABLE_TARGET_OPENMP)
+#if defined( CAMP_ENABLE_OPENMP ) && defined(_OPENMP)
+#define CAMP_HAVE_OPENMP
+#endif
+
+#if defined(CAMP_ENABLE_TARGET_OPENMP)
```

Comment:
Do we need to automatically enable `CAMP_ENABLE_OPENMP` if `CAMP_ENABLE_TARGET_OPENMP` is defined? I was trying to look for this in the top level CMakelists.txt, but didn't see it.

Reply:
I think that specific case is covered in BLT, but what isn't is tying it to the CAMP_ prefixed variable, this definitely needs a fix, but I'll have to pick at it a bit to figure out exactly what.  Really good catch @rchen20, thanks!

Reply:
I added a check for this.  Currently, it's a fatal error to request target OpenMP and not OpenMP in camp.  May decide to adjust that but it seems like a good start.